scripts/docker-run.sh

```diff
@@ -31,6 +31,7 @@ test "$(id -n)" = 1001 ||
     >&2 printf "Warning: this script should be run as user ID 1001 to match the container\n"
 
 set -x
+echo SOF_DOCKER_RUN: $SOF_DOCKER_RUN
```

Comment:
The `set -x` should print the command below which contains $SOF_DOCKER_RUN

Reply:
This is a hack and dropped now.